knapsack_problem/Python/knapsack.py

```diff
@@ -0,0 +1,63 @@
+"""
+Here Knapsack problem has been implemented using Greedy Approach
```

Comment:
Put all these stuff inside the function `knapsack` at near https://github.com/NITSkmOS/Algorithms/pull/372/files#diff-d55ba9556072feccf7fba4e071907fc2R12